### PR TITLE
CTS lab improvements

### PR DESCRIPTION
**Pull request by kgaillot**
Most importantly, this has a fix to handle clusters with a Corosync token timeout above 1 second. Additionally it has a refactor that's been bugging me for a while.

### CODE REVIEW

**Comment by kgaillot** [comment]
@wenningerk , can you review this when you get a chance? No rush

File: cts/lab/CTSscenarios.py [hunk 1]
```diff
@@ -4,7 +4,11 @@
 __copyright__ = "Copyright 2000-2020 the Pacemaker project contributors"
```
Comment (wenningerk) [inline cts/lab/CTSscenarios.py:4]: Is there a strategy when we update the copyright-date? Or every time we touch a file?

**Comment by wenningerk** [comment]
Looking all reasonable.
Assuming moving around all the code is basically cut&paste so that syntax-check on executing assures well enough nothing went wrong there.

File: cts/lab/CTSscenarios.py (hunk 1 above)
Comment (kgaillot) [reply cts/lab/CTSscenarios.py:4]: Whoops, every time -- fixed

**Comment by kgaillot** [comment]
> Assuming moving around all the code is basically cut&paste so that syntax-check on executing assures well enough nothing went wrong there.

Yep

**Pull request merged**